ui-v2/app/templates/components/healthcheck-info.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if (and (lt passingCount 1) (lt warningCount 1) (lt criticalCount 1) )}}
+  <span title="No Healthchecks" class="zero">0</span>
+{{else}}
+  <dl>
+      {{healthcheck-status width=passingWidth name='passing' value=passingCount}}
+      {{healthcheck-status width=warningWidth name='warning' value=warningCount}}
+      {{healthcheck-status width=criticalWidth name='critical' value=criticalCount}}
+  </dl>
+{{/if}}
```

Comment:
New `healthcheck-info` component that composes 3 new `healthcheck-status` now that this is being used in more places

Visually it's this:

<img width="135" alt="screenshot 2019-02-08 at 15 36 27" src="https://user-images.githubusercontent.com/554604/52488258-56654e00-2bb7-11e9-8964-91e5bc0fd7bf.png">